benchmarks/README.md

```diff
@@ -0,0 +1,104 @@
+# WIP - Benchmarking testsuite
+
+The NH benchmark test suite builds on top Nighthawk's integration test framework, and
+can be used to scaffold tests and obtain latency reports as well as flamegraphs.
+
+## Immediate goals
+
+The goal is te be able to:
+- run the suite against arbitrary Envoy revisions
+- persist profile dumps, flamegraphs, and latency numbers per test
+- run the nighthawk tools via docker
+- offer stock tests, but also allow scaffolding consumer-specific tests
```

Comment:
Maybe point out that this is also intended for the use in A/B testing by developers and integration with GitHub CI flows (via the Docker support).